marklogic-data-hub-central/ui/src/App.test.tsx

```diff
@@ -55,4 +55,33 @@ describe('App component', () => {
 
   });
 
+    test('Verify that footer is present in tiles route', async () => {
+        history.push('/tiles');
+        const { getByTestId } = render(<Router history={history}>
+            <AuthoritiesContext.Provider value={mockDevRolesService}>
+                <UserContext.Provider value={userAuthenticated}><App/></UserContext.Provider>
+            </AuthoritiesContext.Provider>
+        </Router>);
+        expect(getByTestId("footer")).toBeInTheDocument();
+    });
+
+    test('Verify that footer is present in no response route', async () => {
+        history.push('/tiles/noresponse');
+        const { getByTestId } = render(<Router history={history}>
+            <AuthoritiesContext.Provider value={mockDevRolesService}>
+                <UserContext.Provider value={userAuthenticated}><App/></UserContext.Provider>
+            </AuthoritiesContext.Provider>
+        </Router>);
+        expect(getByTestId("footer")).toBeInTheDocument();
+    });
+
+    test('Verify that footer is present in error', async () => {
+        history.push('/error');
+        const { getByTestId } = render(<Router history={history}>
+            <AuthoritiesContext.Provider value={mockDevRolesService}>
+                <UserContext.Provider value={userAuthenticated}><App/></UserContext.Provider>
+            </AuthoritiesContext.Provider>
+        </Router>);
+        expect(getByTestId("footer")).toBeInTheDocument();
+    });
```

Comment:
can you pls add a test for "/" aka login component. If possible can you try to get these in one test, alternating the history.push and expect assertion for every pathname.